spec.md

```diff
@@ -277,16 +281,16 @@ The following example shows a CloudEvent serialized as JSON:
 
 ``` JSON
 {
-    "cloudEventsVersion" : "0.1",
-    "eventType" : "com.example.someevent",
+    "cloud_events_version" : "0.1",
+    "event_type" : "com.example.someevent",
     "source" : "/mycontext",
-    "eventID" : "A234-1234-1234",
-    "eventTime" : "2018-04-05T17:31:00Z",
-    "comExampleExtension1" : "value",
-    "comExampleExtension2" : {
-        "otherValue": 5
+    "event_id" : "A234-1234-1234",
+    "event_time" : "2018-04-05T17:31:00Z",
+    "com_example_extension_1" : "value",
```

Comment:
`com_example_extension_1` has 23 characters 😉

Reply:
See!

Reply:
Our current spec metadata names are not consistent, some starts with "event", others not. Better to make them consistent